GREEDY_WEB/ClientApp/components/Home.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { RouteComponentProps } from 'react-router';
+import { ImageUpload } from './ImageUpload';
+import { FetchData } from './FetchData';
+
+interface State {
+    itemList: any
+}
+
+export class Home extends React.Component<RouteComponentProps<{}>, State> {
+    constructor() {
+        super();
+
+        this.getItemList = this.getItemList.bind(this);
+        this.state = {
+            itemList: []
+        };
+    }
+
+    public getItemList(items) {
```

Comment:
change this to     
public getItemList = (items) => { 
and then delete declaration in constructor